stylesheets/layout/navbar.css

```diff
@@ -0,0 +1,87 @@
+@import url(../stylesheets/base/root-variables.css);
+@import url(../stylesheets/base/reset.css);
+
+.navbar{
+   top: 0;
+   z-index: 3;  
+   display: flex;
+   justify-content: space-between;
+   align-items: center;
+   /* flex-direction: row; */
+   padding: 2rem 0.3rem;
+   width: 100%;
+   height: 3.7rem;
+   border-bottom: 2px solid var(--dark-primary-color);
+   box-shadow: var(--box-shadow-btn);
+}
+
+
+.left-nav{
+    display: flex;
+    justify-content: space-evenly;
+    gap: .5rem;
+    align-items: center;
+ }
+
+/* .left-nav{ */
+   /* font-size: 1.7rem; */
+   /* font-weight: bold; */
+   /* color: var(--logo-color); */
+/* } */
```

Comment:
This too can be removed , if not necessary

Reply:
yeah will remove all the commented and unnecessary lines!